From c6df7a98904fc99abc642a5c9f59eaf04d830947 Mon Sep 17 00:00:00 2001
From: Eric Ruei <e-ruei1@ti.com>
Date: Wed, 28 Mar 2018 09:31:46 -0400
Subject: [PATCH 209/210] arm: dts: am571x-idk-pps: add ptp_bc pps
 output-selection gpios

A gpio-controlled multiplexer is added to guarantee that there is only
one PPS output at any time at the AM571(2)x-idk.

This patch add PTP_BC-controlled gpio entries

Signed-off-by: Eric Ruei <e-ruei1@ti.com>
---
 arch/arm/boot/dts/am571x-idk-pps.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/am571x-idk-pps.dts b/arch/arm/boot/dts/am571x-idk-pps.dts
index b367bfd..4750aab 100644
--- a/arch/arm/boot/dts/am571x-idk-pps.dts
+++ b/arch/arm/boot/dts/am571x-idk-pps.dts
@@ -127,5 +127,8 @@
 };
 
 &ptp_bc {
+	pps-sel0-gpios = <&gpio1 4 GPIO_ACTIVE_HIGH>;
+	pps-sel1-gpios = <&gpio6 7 GPIO_ACTIVE_HIGH>;
+
 	status = "okay";
 };
-- 
1.9.1

